ultimate-pipeline/src/ultimate_pipeline/pipelines/data_processing/supervisely_downloader.py

```diff
@@ -0,0 +1,47 @@
+import supervisely as sly
+import typing as t
+
+import os
+
+from dotenv import load_dotenv
+from PIL import Image
```

Comment:
Note: I suggest adding all direct dependencies (i.e. those that we consume directly in code) to requirements.txt.
In PR https://github.com/marcwagn/ultimate_analytics/pull/4, I explain the proposed process.
No action needed in this PR, we can update the requirements files with `PIL` and `tqdm` after the other PR is merged.

Reply:
sounds good. is there an automated way to extract all direct dependencies from pip freeze?

Reply:
Nope, pip itself doesn't store information which packages were installed explicitely (in contrast to e.g. apt in Debian/Ubuntu). And if you uninstall a package that you no longer need, it will most likely leave its dependencies behind - that's why relying on `pip freeze` isn't a good idea, as it can bring too much garbage.

So long story short - after you update your branch with the latest changes from `main`, you can put the following entries to the `requirements.txt` file:
```
pillow==10.2.0
python-dotenv==1.0.0
tqdm==4.66.2
```
No need to update `requirements.lock`, as it already lists those dependencies (as transitive ones).

Reply:
ok thanks. :) done! If the pipeline in image_download works for you as well the download image branch can be merged to main.  I think this feature is done.